src/sendgrid.lisp

```diff
@@ -76,6 +76,9 @@ The JSON looks like:
                         reply-to
                         subject
 			send-at
+		        (attachments nil)
```

Comment:
It's already NIL by default ;)

Reply:
(and below too)

Reply:
I like to be explicit in cases where changing a var has a significant impact. It works :D